### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by Tejinder15**
# Features Added
## I Will Add Quantity Handling Afterwards

### CODE REVIEW

**Review by Veekshith-Thoram — APPROVED** [review]

File: src/Reducer/cart-reducer.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const cartReducer = (state,action) => {
+    switch(action.type){
+        case "Get_Cart":
```
Comment (Ayushi20-19) [inline src/Reducer/cart-reducer.js:3]: it is good practice to name your action type as uppercase like GET_CART

**Review by Ayushi20-19 — APPROVED** [review]
Code looks good 
The file structure also seems good 
Try to name action type as uppercase

File: src/Components/PriceCard/PriceCard.jsx [hunk 2]
```diff
@@ -1,34 +1,46 @@
...
+      <hr />
+      <div className={styles.total_amount_container}>
+        <span className={styles.total_amount_title}>Total Amount</span>
+        <span className={`${styles.total_amount} price`}>
+          &#8377;{props.totalprice + 120 - 500}
+        </span>
+      </div>
+      <hr />
+      <p>
+        You will save{" "}
```
Comment (Ayushi20-19) [inline src/Components/PriceCard/PriceCard.jsx:35]: Remove {" "}..

Comment (Tejinder15) [reply src/Components/PriceCard/PriceCard.jsx:35]: It is given by React itself not me it shows up even if i remove it.

**Pull request merged**